app/scripts/directives/overview/listRow.js

```diff
@@ -150,6 +151,8 @@
 
     row.canIDoAny = function() {
```

Comment:
So the purpose of this function seems to be "if I can do anything at all, show the dropdown".  Its fairly lengthy, and a lot of the logic is repeated in the template with all the `ng-if` statements.  I'm wondering if there is a way to break it down?  

I pause at the `switch` with nested `if` that have `//comments` that are needed to reveal the true intention.  I wonder if the details of this could be broken up into smaller functions and/or pushed up into the related `Pod`, & `Deployment` services (ex:`DeploymentService.canIDoAny()`).